src/write_gcm.c

```diff
@@ -3568,15 +3568,20 @@ static void *wg_process_queue(wg_context_t *ctx, wg_queue_t *queue,
     }
     if (wg_update_stats(stats) != 0) {
       wg_some_error_occured_g = 1;
-      WARNING("%s: wg_update_stats failed.", this_plugin_name);
+      ERROR("%s: wg_update_stats failed.", this_plugin_name);
       break;
     }
     payloads = NULL;
```

Comment:
@kosak : Corey - I was wondering if you had any specific motivation for setting the pointer to NULL as it is anyway going out of scope.?